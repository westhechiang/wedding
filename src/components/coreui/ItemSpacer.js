import styled from 'styled-components';

const ItemSpacer = styled.div`
  height: ${props => props.height || '100%'};
  width: ${props => props.width || 'auto'};
  margin-right: ${props => (props.space || '10px')};
  position: ${props => (props.position || 'relative')};
  &:last-child {
    margin-right: 0;
  }
`;

export default ItemSpacer;
