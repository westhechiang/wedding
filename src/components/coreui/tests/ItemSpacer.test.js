import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ItemSpacer from '../../../components/coreui/ItemSpacer';

describe('<ItemSpacer />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<ItemSpacer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
