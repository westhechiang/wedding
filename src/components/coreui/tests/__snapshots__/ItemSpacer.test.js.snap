// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemSpacer /> should match its empty snapshot 1`] = `
<div
  className="sc-bdVaJa jlOtIu"
/>
`;
